speech_decoding/constants.py

```diff
@@ -0,0 +1,13 @@
+import torch, random
+import numpy as np
+
+# assert torch.cuda.is_available(), "Training without GPU is not supported."
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+BAR_FORMAT = "{desc:<5.5}{percentage:3.0f}%|{bar:10}{r_bar}"
+
+# NOTE: I'm using GPU on my machine so heavily on other project that I can't use
+# device = "cpu"
+
+BRAIN_RESAMPLE_RATE = 120
```

Comment:
Why were these added to `constants.py` instead of being in the config as before?

Reply:
I though those that are not hyperparameters and shouldn't be changed could be in `constants.py`?

Reply:
I see. I think in that case it should be OK (Python doesn't have any recommended way of dealing with this and `constants.py` is one solution) - but if the code can be run on CPU then definitely `DEVICE` should not be there and code should be device-agnostic.